doc/dev/bin/pip_download_test.sh

```diff
@@ -41,7 +45,12 @@ do
     printf "\n\n\n"
 
     pip install redis==3.3.2
-    pip install msgpack==0.6.2
+    pip install msgpack==1.0.0
```

Comment:
cc @barakmich When we push the wheel to test repo of pypi, the wheel doesn't contain all dependencies, so we should manually download them here.

Reply:
What gives? The wheel should at least declare its dependencies in setup.py, no?

Reply:
^ @rkooo567

Reply:
I am not sure about the exact details. cc @simon-mo

Reply:
This section is about testing ray wheels from a test PyPI instance. The test PyPI instance is a standalone pip package repository for that doesn't contains our deps.

Reply:
More detail in step 8 of doc/dev/release_process.rst